faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -257,7 +257,7 @@ object Encoder {
   }
 
   implicit object InstantEncoder extends Encoder[Instant] {
-    override def encode(t: Instant): Value = TimeV(t)
+    override def encode(t: Instant): Value = if (t != null) TimeV(t) else NullV
```

Comment:
Interesting. No other encoder have a similar if, why is this one special?

Reply:
LocalDate and String do, just below this in the same file.